DEVELOPMENT.md

```diff
@@ -0,0 +1,328 @@
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/traceability/v1"
+  ],
+  "id": "https://example.com/123",
+  "type": ["VerifiableCredential"],
+  "issuer": {
+    "id": "did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM"
+  },
+  "issuanceDate": "2020-03-10T04:24:12.164Z",
+  "expirationDate": "2029-03-10T04:24:12.164Z",
+  "credentialSubject": {
+    "id": "did:example:123",
+    "type": "Place",
+    "address": {
+      "type": "PostalAddress",
+      "organizationName": "Bednar - Kutch",
+      "streetAddress": "8440 Khalid Canyon",
+      "addressLocality": "Gislasonland",
+      "addressRegion": "Ohio",
+      "postalCode": "96546",
+      "addressCountry": "Liechtenstein"
+    }
+  },
+  "proof": {
+    "type": "Ed25519Signature2018",
+    "created": "2021-02-09T21:54:29.223Z",
+    "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..1Xba7-B-YTstwnbIewQFpbMgGfOdRuoDLEBezZlb4M1qMwI9GCUAbeTCsGCPd62XLwyNjPqb9aJGj_xk7iUiBw",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM#z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM"
+  }
+}
+```
+
+Inside JSON Schema, you will see things like —
+
+```json
+ "$ref": "https://w3id.org/traceability/schemas/Product.json"
+```
+
+These are JSON Schema reference tags, and they allow for nesting of types that
+are defined in JSON Schema.
+
+In general, you should strive to model concepts as types that are defined in
+JSON Schema and composed from smaller types.
+
+## Versioning
+
+This repository will be versioned at periodic points in time with a Q1 Calendar
+Year target for major releases. Versioning tags will follow a pattern of
+`[MAJOR].[MINOR].[PATCH]`, defined as follows:
+
+- `MAJOR` - significant changes rolled forward from the previous major version.
+  Major versions MAY include breaking or non-backwards compatible changes.
+- `MINOR` - backwards compatible changes that may introduce new functionality
+  or extensions of objects that are backwards compatible
+- `PATCH` - minor changes that are non-breaking and resolve discovered issues
+  or bugs
+
+As a rule, versioning will follow the specification outlined in the [Semantic
+Versioning 2.0](https://semver.org/) spec.
+
+This approach to versioning gives the ability to integrate with provided
+automated testing and validation against defined types without worry of
+instability or breaking changes being introduced, while also limiting the
+frequency of possibly breaking changes to prevent a large number of incompatible
+versions.
+
+## Project Structure
+
+This project uses [lerna](https://github.com/lerna/lerna) to manage packages and
+dependencies as a single project.
+
+In addition, the following key areas in the repo should be noted for
+understanding how to add schemas and understand the code layout:
+
+- The index file in [docs](./docs/index.html) is the master public-facing
```

Comment:
I believe this may be outdated? 
If so lets create an issue for it before merging the PR

Reply:
https://github.com/w3c-ccg/traceability-vocab/issues/244